examples/marketing-campaign/src/marketing_campaign/app.py

```diff
@@ -27,13 +25,19 @@
 EMAIL_REVIEWER_CONFIG = ApiClientConfiguration.fromEnvPrefix("EMAIL_REVIEWER_")
 
 # Set to True to generate a mermaid graph
-GENERATE_MERMAID_GRAPH = os.environ.get("GENERATE_MERMAID_GRAPH", "False").lower() == "true"
+GENERATE_MERMAID_GRAPH = (
+    os.environ.get("GENERATE_MERMAID_GRAPH", "False").lower() == "true"
+)
 
 
-def process_inputs(state: state.OverallState, config: RunnableConfig) -> state.OverallState:
-    cfg = config.get('configurable', {})
+def process_inputs(
+    state: state.OverallState, config: RunnableConfig
+) -> state.OverallState:
+    cfg = config.get("configurable", {})
 
     user_message = state.messages[-1].content
+    state.recipient_email_address = cfg["recipient_email_address"]
```

Comment:
The "recipient_email_address" is a required key, but in the previous line the dict is allowed to be empty. Is this key also optional, really? Same as the sender in the next line.